libs/features/texts/src/broadcast-text/broadcast-text.component.html

```diff
@@ -61,6 +89,13 @@
             </div>
             <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
               <label class="control-label" for="EmoteID3">EmoteID3</label>
+              <keira-single-value-selector-btn
+                [control]="editorService.form.controls.EmoteID3"
+                [disabled]="editorService.form.controls.EmoteID3.disabled"
+                [config]="{ options: EMOTE, name: 'emote' }"
+                [modalClass]="'modal-md'"
+              />
+
```

Comment:
maybe add an integration test for each modal type? see how other existing integration tests check the modals